src/sentry/api/endpoints/project_users.py

```diff
@@ -42,3 +43,30 @@ def get(self, request, project):
             paginator_cls=DateTimePaginator,
             on_results=lambda x: serialize(x, request.user),
         )
+
+    def delete(self, request, project):
+        """
+        Delete an Event User
+        ````````````````````````````````
+
+        Delete an event's user.
+
+        :pparam string organization_slug: the slug of the organization.
+        :pparam string project_slug: the slug of the project.
+        :qparam string query: Prefix ``id`` should be used to match on.
+                              For example, ``query=id:3``
+        """
+        if is_active_superuser(request):
```

Comment:
If the user isn't a superuser should we raise an error? Right now the client would get a 200 which isn't what actually happened as nothing was deleted.